src/test/scala/no/ndla/articleapi/UnitSuite.scala

```diff
@@ -28,7 +28,7 @@ abstract class UnitSuite extends FunSuite with Matchers with OptionValues with I
 
   setEnv("NDLA_BRIGHTCOVE_ACCOUNT_ID", "some-account-id")
   setEnv("NDLA_BRIGHTCOVE_PLAYER_ID", "some-player-id")
-  setEnv("SEARCH_ALIAS", "integration-test-index")
```

Comment:
Bruker du ikke nå samme navn for integrasjonstester i forskjellige repo'er? Vil ikke da disse kunne gå i beina på hverandre når du kjører testene mot samme elasticsearch-engine? (Tilsvarende argumentasjon som @kstigen hadd for at vi ikke ønsker å bruke `articles` som navn både for kjørende kode og testkode)